src/hooks/useArbTokenBridge.ts

```diff
@@ -277,11 +276,7 @@ export const useArbTokenBridge = (
   }
 
   const withdrawToken = async (erc20l1Address: string, amount: string) => {
-    const tokenData = (await bridge.getAndUpdateL1TokenData(erc20l1Address))
-      .ERC20
-    if (!tokenData) {
-      throw new Error("Can't withdraw; token not found")
-    }
+    const tokenData = await bridge.l1Bridge.getL1TokenData(erc20l1Address)
     const amountParsed = utils.parseUnits(amount, tokenData.decimals)
```

Comment:
Same as above, if the UI provides the parsed amount we can save on these network queries.